aws/logs_monitoring/parsing.py

```diff
@@ -246,7 +246,13 @@ def find_cloudwatch_source(log_group):
         return "rds"
 
     # e.g. Api-Gateway-Execution-Logs_xxxxxx/dev
-    if log_group.startswith("api-gateway"):
+    #      /aws/api-gateway/my-project
+    #      /aws/http-api/my-project
+    if log_group.startswith((
```

Comment:
mind leaving a comment on each line for the type of logs being addressed? 

1. default location for rest api execution logs
2. default location set by serverless framework for rest api access logs
3. default location set by serverless framework for http api logs

Reply:
Sure! 👍